ontology/mp.md

```diff
@@ -43,6 +43,12 @@ browsers:
   - label: Monarch
     title: Monarch Phenotype Page
     url: http://monarchinitiative.org/phenotype/MP:0000001
+usages:
```

Comment:
@sbello can you digg out other usages, like IMPC, in the same way as we did this one here? The more the better!

Reply:
Now that the formatting issue is fixed I'll add some additional usages